Как правильно расположить фрагмент "Игра 1"?

georgeheqd opened the pull request:
Не получается расположить фрагмент "Игра 1" как на макете сайта

georgeheqd committed:
Как правильно расположить фрагмент "Игра 1"?

georgeheqd committed:
Проблема с заданиями 9.4 - 10

georgeheqd committed:
Как сделать 9.4 ?

georgeheqd committed:
Как сделать задание 9.4?

izvolte commented on hw_1.js:
```diff
@@ -0,0 +1,44 @@
+a += 10;
+alert(a)*/
+
+/*let age = "29.06.2007";
```
в названии переменной нет никакой инфомации о содержании

izvolte commented on hw_1.js:
```diff
@@ -0,0 +1,44 @@
+
+/*let age = "29.06.2007";
+alert("Дата выходы первого айфона " + age) ; */
+/*let Name = "Брендан Айк"
```
в названии переменной нет никакой инфомации о содержании

izvolte commented on hw_1.js:
```diff
@@ -0,0 +1,44 @@
+num *= 7;
+num /= 3;
+num ++;
+num --;
```
лучше такое без пробелом 

num--
num++

georgeheqd commented:
"в названии переменной нет никакой инфомации о содержании" - мне нужно просто вместо "a" другое название придумать для контейнера ?

georgeheqd commented:
И как правильно сделать 9.4 ?

georgeheqd committed:
Проблема с mobile.scss

georgeheqd committed:
Не работает адаптив на мобилки

georgeheqd committed:
Add files via upload

Не работает адаптивная часть, не понимаю что нужно изменить, а так же в теге body меняется сильно ширина в 2 или 3 раза.

georgeheqd committed:
Небольшие исправления

georgeheqd committed:
Адаптация

georgeheqd committed:
Merge branch 'hw1' of https://github.com/georgeheqd/2-nd-course-hw into hw1

georgeheqd committed:
hw_3

Не получилось 1 и 6 задание по js. Как правильно настроить анимацию наведения?

georgeheqd committed:
Не работает эффект наведения на карточки

georgeheqd committed:
Не получился эффект наведения на корточках

georgeheqd committed:
Не получился эффект наведения на корточках

georgeheqd committed:
Не получился эффект наведения на корточках

georgeheqd committed:
Не понял как улучшить 5 задание

georgeheqd committed:
Не совсем разобрался как сделать 6 задание

izvolte commented on hw_4.js:
```diff
@@ -0,0 +1,50 @@
+		break; // прекращвем выполнение цикла
+	}
+}
+while (true) console.log('Привет!');
```
Нужно ведь всего 2 раза, а не бесконечное количество раз

izvolte commented on hw_4.js:
```diff
@@ -0,0 +1,50 @@
+while (i < 5) {
+	console.log(i);
+	i++;
+}
```
а где же 3-е задание?

izvolte commented on hw_4.js:
```diff
@@ -0,0 +1,50 @@
+	if (n /=2 ) {
+		continue;
+	}
+	sum++
```
n /= 2
	sum++

так достаточно

izvolte commented on hw_4.js:
```diff
@@ -0,0 +1,50 @@
+	console.log("Сегодня пятница, " + dayNumber + "-е число. Необходимо подготовить отчет.")
+}
+for (let dayNumber = 1; dayNumber <= 31; dayNumber++) {
+    if (dayNumber == 5) {
```
в целом можно делить на 5 и если нет остатка то уже выводить число

попробуй переделать

georgeheqd replied:
Если делить на 5 то там же во всех почти что случаях будет остаток разный или я не так понял?

izvolte replied:
Да ты прав

В таком случае мы определяем переменную

firstFriday = 3 // к примеру

for (let dayNumber = firstFriday; dayNumber <= 31; dayNumber += 7) { // сразу прибавляем 7 чтобы экономить итерации

и из-за того что мы каждый раз прибавляем 7

нам достаточно каждую итерацию ввыодить в консоль

izvolte replied:
for (let dayNumber = firstFriday; dayNumber <= 31; dayNumber += 7) {
        console.log("Сегодня пятница, " + dayNumber + "-е число. Необходимо подготовить отчет.");
}

без всяких if'ов

georgeheqd committed:
Переделал остальные задания кроме 6

georgeheqd committed:
переделал, но теперь он бесконечно прибавляет 7

georgeheqd committed:
Не получается 1 задание и 3

izvolte commented on hw_5.js:
```diff
@@ -0,0 +1,78 @@
+ let b = prompt('Число');
+
+
+ console.log(comp());
```
осталось только передать аргументы

 console.log(comp(a, b));

izvolte commented on hw_5.js:
```diff
@@ -0,0 +1,78 @@
+        return b;
+    }
+    else if (a == b) {
+        return a || b;
```
тут всегда будет отдавать 'a' если она не 0

1 || 10 = 1

10 || 1 = 1

0 || 10 = 10

Я предлагаю просто сделать


    if (a => b) {
        return a;
    }
    else if (a < b) {
        return b;
    }

izvolte commented on hw_5.js:
```diff
@@ -0,0 +1,78 @@
+
+    printMessage = function () {
+        console.log("Четное");
+    }
```
условие должно быть внутри функции
```

function checkNumber(num) {
     
  if (numb % 2 == 0) {
          console.log("Четное");
  } else {
          console.log("Нечетное");
  }
}

checkNumber(promt("Число"))
```
